tests/reader-errors.rs

```diff
@@ -14,23 +14,27 @@ macro_rules! ok {
             fn borrowed() {
                 let mut reader = Reader::from_str($xml);
                 reader.config_mut().enable_all_checks(true);
-                assert_eq!(reader.read_event().unwrap(), $event);
+                assert_eq!(reader.read_event().unwrap(), $event, "Reader");
```

Comment:
Given this is only a few dozen lines of tests, why not just split it up into individual tests under a `mod reader` and `mod ns_reader`?